pkg/parse/run.go

```diff
@@ -234,6 +241,12 @@ func run(ctx context.Context, p Parser, trigger string, state *reconcilerState)
 // It also updates the .status.rendering and .status.source fields.
 func read(ctx context.Context, p Parser, trigger string, state *reconcilerState, sourceState sourceState) status.MultiError {
 	hydrationStatus, sourceStatus := readFromSource(ctx, p, trigger, state, sourceState)
+	if status.HasTransientErrors(hydrationStatus.errs) {
```

Comment:
Add a comment explaining why it's desirable and safe to fail due to transient errors. It's not immediately obvious from the code.

Reply:
done